android/app/src/main/java/app/organicmaps/util/ThemeSwitcher.java

```diff
@@ -32,20 +32,17 @@ public void run()
       String nightTheme = MwmApplication.from(mContext).getString(R.string.theme_night);
       String defaultTheme = MwmApplication.from(mContext).getString(R.string.theme_default);
       String theme = defaultTheme;
+      Location last = LocationHelper.from(mContext).getSavedLocation();
 
-      if (RoutingController.get().isNavigating())
+      if (last == null)
       {
-        Location last = LocationHelper.from(mContext).getSavedLocation();
-        if (last == null)
-        {
-          theme = Config.getCurrentUiTheme(mContext);
-        }
-        else
-        {
-          boolean day = Framework.nativeIsDayTime(System.currentTimeMillis() / 1000,
-                                                  last.getLatitude(), last.getLongitude());
-          theme = (day ? defaultTheme : nightTheme);
-        }
+        theme = Config.getCurrentUiTheme(mContext);
+      }
```

Comment:
Nit: we don't use braces for one-liners:

```java
if (last == null)
  theme = Config.getCurrentUiTheme(mContext);
```

Reply:
done 👍️